src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -66,7 +66,7 @@ public class Startup
             Based on the target framework
          */
 
-        private static readonly string DefaultTargetFramework = "net6.0";
+        private static readonly string DefaultTargetFramework = "net7.0";
```

Comment:
Really? `net6.0` is going to be supported longer than 7. Do we just want to jump to 8?

Reply:
It's just if we don't set any framework it will use `net7.0`, once `net8.0` is released officially we can switch. Note that it is never used in our benchmarks since we always set it explicitly.